### PR TITLE
Docker: simplify build script

### PR DESCRIPTION
**Pull request by hainest**

### CODE REVIEW

File: docker/build.sh [hunk 1]
```diff
@@ -6,33 +6,40 @@ set -euo pipefail
...
-make install VERBOSE=1 -j2 > >(tee build-install.out) 2> >(tee build-install.err >&2)
+echo "::group::build dyninst"
+
+DYNINST_BUILD_DIR=/opt/dyninst-env/build/dyninst
+mkdir -p $DYNINST_BUILD_DIR
+
+DYNINST_INSTALL_DIR=/opt/dyninst-env/install/dyninst
+mkdir -p $DYNINST_INSTALL_DIR
+
+cmake -S /code -B $DYNINST_BUILD_DIR -DCMAKE_INSTALL_PREFIX=$DYNINST_INSTALL_DIR
```
Comment (vsoch) [inline docker/build.sh:20]: You'll probably want to make groups around here (see the part you deleted below) so the output is meaningfully organized in the action.

Comment (hainest) [reply docker/build.sh:20]: Sorry, I don't follow. The beginning of the dyninst build group is on line 12 and ends on 24.

Comment (vsoch) [reply docker/build.sh:20]: Sorry I’m just blind didn’t see it. :)

Comment (hainest) [reply docker/build.sh:20]: No worries. I need someone to double-check! I'm still feeling my way through this.

**Comment by hainest** [comment]
_Finally_.

**Pull request merged**